utils/controleval.py

```diff
@@ -92,6 +92,12 @@ def get_product_yaml(product):
     exit(1)
 
 
+def get_formatted_name(raw_string):
+    for special_char in '-. ':
+        canonical_name = raw_string.replace(special_char, '_')
+    return canonical_name
```

Comment:
This probably doesn't do what you want because it replaces only spaces because the canonical_name variable is overwritten during iterations.

Reply:
You are correct @jan-cerny . I initially used different names in this function during the development and forgot to finish the changes before sending the commit. I haven't noticed since the current status only contain spaces but this could be an issue in the future. Good that you caught it. Thanks. It is already fixed.